packages/terra-dropdown-button/tests/jest/__snapshots__/SplitButton.test.jsx.snap

```diff
@@ -105,7 +105,11 @@ exports[`Dropdown Button correctly applies the theme context className 1`] = `
             onKeyUp={[Function]}
             type="button"
           >
-            Primary Option
+            <span
+              className=""
```

Comment:
Not sure if we want to leave the className attribute blank here or just remove it entirely.

Reply:
NIT: I'd probably removed it for cleaner look, unless I am missing something. What is a benefit of keeping it?

Reply:
No benefit, and I agree with you on the cleaner look. I'll remove it.

Reply:
removed: [def249f](https://github.com/cerner/terra-core/pull/4080/commits/def249fad0186f93ce946799f1eff6bc01c74449)